### PR TITLE
Request to merge feature branch

### PR DESCRIPTION
**Pull request by kimothokamau**
- Created functions to achieve users ability to add and remove books
- Data persists in the localstorage

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+const title = document.getElementById('title');
+const author = document.getElementById('author');
+const form = document.querySelector('#form');
+const bookContainer = document.getElementById('book-container');
+// const addBtn = document.getElementById('addBtn');
```
Comment (mricanho) [inline main.js]: - [x] Remove all commented code

File: main.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+// let rmvBtn;
+
+// console.log(addBtn);
```
Comment (mricanho) [inline main.js]: - [x] Remove all commented code

File: main.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+}
+
+function createBook() {
+  const bookDiv = document.createElement('div');
+  const bookTitle = document.createElement('h3');
+  bookTitle.textContent = title.value;
+  bookDiv.append(bookTitle);
+  const bookAuthor = document.createElement('h3');
+  bookAuthor.textContent = author.value;
+  // bookContainer.append(bookTitle,bookAuthor)
```
Comment (mricanho) [inline main.js]: - [x] Remove all commented code

File: main.js [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+  bookDiv.append(bookAuthor);
+  const removeBtn = document.createElement('button');
+  removeBtn.className = 'remv-cls';
+  removeBtn.textContent = 'remove';
+  bookDiv.append(removeBtn);
+  bookContainer.append(bookDiv);
+  // rmvBtn = document.querySelector('.remv-cls');
```
Comment (mricanho) [inline main.js]: - [x] Remove all commented code

File: main.js [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+  //     console.log(eachIetndex)
+  //     // e.target.parentElement.remove()
+  //     // //displayBooks(myLibrary[myLibrary.length-1])
+
+  //     // myLibrary.forEach((book, index) => {
+  //     //   if (index === eachIndex) {
+  //     //     myLibrary.splice[eachIndex, 1]
+  //     //   }
+  //     // })
+  //   }
```
Comment (mricanho) [inline main.js]: - [x] Remove all commented code

File: main.js [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+// let rmvBtn;
+
+// console.log(addBtn);
+
+const library = [];
+function Book(title, author) {
+  this.title = title;
+  this.author = author;
+}
+// localStorage.setItem('books',library)
```
Comment (mricanho) [inline main.js]: - [ ] You need to add a function to preserve data with local storage

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @kimothokamau @Jmagero, 

Good job so far! 🔥🔥 This is a very good first attempt, many of your JS code are functional and you are following the gitflow.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

- [ ] All linters must pass, right now the eslint linter is failing, please fix it.
- [ ] Data must be preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md)

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

you can do it

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kimothokamau** [comment]
- Removed commented code
- Add localstorage persistence

File: main.js (hunk 1 above)
Comment (henatan99) [reply main.js]: This issue is well addressed! :clap:

File: main.js (hunk 2 above)
Comment (henatan99) [reply main.js]: This issue is well addressed! :clap:

File: main.js (hunk 3 above)
Comment (henatan99) [reply main.js]: This issue is well addressed: :+1:

File: main.js (hunk 4 above)
Comment (henatan99) [reply main.js]: This one is also addressed!  :dancers:

File: main.js (hunk 5 above)
Comment (henatan99) [reply main.js]: This issue is well addressed! :man_dancing:

File: main.js (hunk 6 above)
Comment (henatan99) [reply main.js]: Nice work is done here. Well addressed!! :champagne:

**Review by henatan99 — APPROVED** [review]
Hi @kimothokamau , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**